src/index.js

```diff
@@ -92,11 +92,11 @@ document.addEventListener('DOMContentLoaded', () => {
   renderTaskList();
 });
 
-// const clearButton = document.getElementById('clear-btn');
-// clearButton.addEventListener('click', () => {
-//   tasks = tasks.filter((task) => !task.completed);
-//   localStorage.setItem('tasks', JSON.stringify(tasks));
-//   renderTaskList();
-// });
+const clearButton = document.getElementById('clear-btn');
+clearButton.addEventListener('click', () => {
```

Comment:
[OPTIONAL] When the completed tasks are cleared, it would be best if you also updated the `index` of each remaining task so they reflect their new respective position in the tasks array.

For example, I deleted the 1st and 2nd task in the following illustration.

![image](https://user-images.githubusercontent.com/38088818/230709951-2f32f096-145e-4bd1-93e7-4271c203db55.png)